README.md

```diff
@@ -20,15 +20,200 @@ tasks specified in the Gruntfile.
+
+As of this writing, by default NodeJS installs its commands into `/usr/local/bin/`. In the **terminal**, type:
+```
+$ ls /usr/local/bin/npm
+```
+If this command outputs something like `/usr/local/bin/npm`, but you are still unable to run:
+```
+$ npm --version
+```
+try running:
+```
+$ /usr/local/bin/npm --version
+```
+If this is successful, then **npm** is successfully installed, and you will just have to add `/usr/local/bin/` to your system *PATH* variable (see below).
+If the command:
+```
+$ ls /usr/local/bin/npm
+```
+outputs a message telling you permission is denied, then you will have to change the ownership of the `/usr/local/` and `/usr/local/bin/` directories. On Mac, follow the [instructions to take ownership](http://osxdaily.com/2013/04/23/change-file-ownership-mac-os-x/) of these directories, or to at least give yourself read permission. On other Unix systems, use the **chown** command or the user-interface appropriate to your distribution to do so.
+
+##### Grunt
+After installing NodeJS, install **grunt** by doing the following:
+**Note:** These installation steps are copied from the [Grunt Getting Started guide](https://gruntjs.com/getting-started).
+On Windows, open a **cmd** window (go to Start Menu, search for **cmd** and open it); on MacOSX, open a **terminal** window. Within this window, type:
+```
+$ npm install -g grunt-cli
+```
+If the above command is unsuccessful, some machines may need to append sudo at the beginning of the command. If **grunt** is successfully installed, the following command:
+```
+$ grunt --version
+```
+Should display the installed version of **grunt**. For example the version might be `grunt-cli v1.2.0`.
+**Warning:** If **grunt** is not found, you may need to add it to the *PATH* variable of your system.
+
+##### Android SDK
+To install the Android SDK:
+1) Browse to the Android SDK download page.
+2) Scroll down on this page to the section labeled: Get just the command line tools.
+   **Note**: You can alternatively install the full Android Studio if you so wish, in which case you should follow Google's instructions and then skip to step 6 of this guide.
+3) Within that section, download the appropriate file(s) based on your operating system.  
+4) Accept the license agreement    
+5) Wait for the install of the SDK Tools to complete. Windows will need to manually run the .exe file previously downloaded to start installation.    
+6) Run the SDK Manager
+   - On Windows, it is available in the `Start Menu` under Android SDK Tools
+   **Warning:** If the packages fail to install, you may need to run the Android SDK as an Administrator.
+   - On Mac/Unix, open the SDK folder you downloaded above. In the `bin/` or `tools/` directories (on some versions it is in both places–it doesn't matter which you use), double click the file called `android`.
```

Comment:
This tool isn't present in the latest command line tools anymore. There is the `sdkmanager` CLI though.

Reply:
@iamhks Steps 7 through 11 also need to be updated. The graphical tool doesn't come with the command line tools but available through the full Android Studio install.

Reply:
@linl33 Does the updated one look good?